tests/nlu/selectors/test_response_selector.py

```diff
@@ -0,0 +1,240 @@
+from pathlib import Path
```

Comment:
Sorry, i didn't see this earlier, but there is already a file named `test_selectors.py` where all of the content of this file can go in. We don't need a separate file.